src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Big from 'big.js';
+
+const operate = (num1, num2, operation) => {
+  const First = Big(num1);
+  const Second = Big(num2);
+
+  switch (operation) {
+    case '+':
+      return First.plus(Second);
+    case '-':
+      return First.minus(Second);
+    case 'x':
+      return First.times(Second);
+    case '/':
+      if (Second === 0) {
+        return 'undefined';
+      }
```

Comment:
Let's handle this better, it should give some explanatory error, like `Can't divide by 0`, `undefined` is kind of vague.